modules/lss-shareAlliancePost/ShareAlliancePost.js

```diff
@@ -0,0 +1,298 @@
+(function (I18n, $) {
+
+    let LSS_SHAREALLIANCEPOST_STORAGE = "LSS_SHAREALLIANCEPOST_STORAGE";
+
+    I18n.translations.de.lssm.sharealliancepost = {
+        share: 'Teilen...',
+        chat: 'Chatten...',
+        alert: 'Alarmieren...',
+        reqVehIdent: 'Zusätzlich benötigte Fahrzeuge:',
+        reqVeh: 'Keine weiteren Fahrzeuge benötigt.',
+        settings: {
+            title: 'Alarmieren, Teilen & Posten',
+            jumpNext: 'Nach Alarmierung zum nächsten Einsatz springen',
+            enableKeyboard: 'Alarmierung mit Tastatur aktivieren',
+            defaultPostToChat: 'Post in Chat als Standard festlegen',
+            shortcutKeys: 'Kommagetrennte Tasten für Tastatur Alarmierung. [17 = ctrl, 89 = y] (<a href="https://keycode.info" target="_blank">Info</a>)',
+            messages: 'Nachrichten Vorlagen. Platzhalter: <ul><li>%ADDRESS%</li><li>%TIME_OFFSET%</li><li>%PATIENTS_LEFT%</li><li>%REQUIRED_VEHICLES%</li></ul>',
+            timeOffset: 'Offset für %TIME_OFFSET%-Formatter in Stunden'
+        }
+    };
+
+    I18n.translations.en.lssm.sharealliancepost = {
+        share: 'Share...',
+        chat: 'Chat...',
+        alert: 'Alert...',
+        reqVehIdent: 'Needed vehicles:',
+        reqVeh: 'No other vehicles required.',
+        settings: {
+            title: 'Alert, Share & Post',
+            jumpNext: 'Jump to next mission after alerting',
+            enableKeyboard: 'Enable Keyboard shortcuts',
+            defaultPostToChat: 'Post to chat is default',
+            shortcutKeys: 'Comma separated list of keys for alerting. [17 = ctrl, 89 = y] (<a href="https://keycode.info" target="_blank">Info</a>)',
+            messages: 'Message templates. Placeholders: <ul><li>%ADDRESS%</li><li>%TIME_OFFSET%</li><li>%PATIENTS_LEFT%</li><li>%REQUIRED_VEHICLES%</li></ul>',
+            timeOffset: 'Offset for %TIME_OFFSET%-formatter in hours'
+        }
+    };
+
+    I18n.translations.nl.lssm.sharealliancepost = {
+        share: 'aandeel...',
+        chat: 'Chat...',
+        alert: 'Alarm...',
+        reqVehIdent: 'Benodigde voertuigen:',
+        reqVeh: 'Geen andere voertuigen nodig.',
+        settings: {
+            title: 'Waarschuwing, delen & posten',
+            jumpNext: 'Spring naar de volgende missie na het alarmeren',
+            enableKeyboard: 'Sneltoetsen op het toetsenbord inschakelen',
+            defaultPostToChat: 'Post om te chatten is standaard',
+            shortcutKeys: 'Kommagescheiden lijst van sleutels om te waarschuwen. [17 = ctrl, 89 = y] (<a href="https://keycode.info" target="_blank">Info</a>)',
+            messages: 'Boodschapssjablonen. Plaatshouders: <ul><li>%ADDRESS%</li><li>%TIME_OFFSET%</li><li>%PATIENTS_LEFT%</li><li>%REQUIRED_VEHICLES%</li></ul>',
```

Comment:
'Bericht voorbeelden. Plaatshouders:

(sorry, I don't really know how to correct translate "plaatshouders")